src/VisualStudio/Core/Def/Options/VisualStudioOptionPersisterProvider.cs

```diff
@@ -61,11 +66,13 @@ private void RefreshOption(OptionKey2 optionKey, object? newValue)
             }
         }
 
-        private async ValueTask<I?> TryGetServiceAsync<T, I>() where I : class
+        private static async ValueTask<I?> TryGetServiceAsync<T, I>(IVsService<T, I> service)
```

Comment:
Do we need this anymore? You removed the use of it for ISettingsManager, but didn't remove the use for the feature flags service, and it's not clear to me why we're being inconsistent that way.

Reply:
➡️ Will clean this up in follow-up